### DISCUSSION TITLE
OCSP URI support for VPN

### DISCUSSION THREAD

**Discussion by apetrelli** [mitmproxy/mitmproxy#6343] [Q&A]
Greetings
I am currently trying to MITM in transparent mode on my local machine the connection to a proprietary VPN.
The thing is that everything works until it tries to do OCSP checking.

Looking at the log, I notice that in the original certificate, the OCSP URI is present, it tries to call it and it fails. But it goes on.

With mitmproxy on, it notices that there is no OCSP URI and only in this case it fails.

Is there a way to put a fake OCSP URI in the certificate? After all, if it fails it goes on.

Thanks in advance.
Antonio Petrelli

**Comment by mhils** [comment]
We don't have a feature specifically for this. However, you can manually generate a certificate that matches your requirements and then use https://docs.mitmproxy.org/stable/concepts-certificates/#using-a-custom-server-certificate. :)
